Header header
float32 neckRotat		# left - right rotation
float32 neckPan         # up - down inclinaison
float32 neckTilt        # left - right inclinaison